moralization/transformers_model_manager.py

```diff
@@ -311,12 +320,11 @@ def load_dataloader(
             batch_size=batch_size,
         )
 
-    def load_optimizer(self, learning_rate: float = 2e-5, kwargs: Dict = None) -> None:
+    def load_optimizer(self, learning_rate: float, kwargs: Dict = None) -> None:
```

Comment:
can this be made private? Seems to be always called by `_initialize_training` when user calls `train` so no need to be public, or is there another use case where it would be called directly by the user? I guess same question for the other public methods called by `_initialize_training`
```suggestion
    def _load_optimizer(self, learning_rate: float, kwargs: Dict = None) -> None:
```